packages/generator/src/sdk-metadata/generation-and-usage.ts

```diff
@@ -42,5 +41,10 @@ export const linkGenerationDocumentaion =
   'https://sap.github.io/cloud-sdk/docs/js/features/odata/generate-odata-client';
 
 export function getGenerationDocumentation(): string {
-  return `Please follow the documentation ${linkGenerationDocumentaion}`;
+  return `<ul>
```

Comment:
I assume you tested the html tags.

Reply:
Yes I copied it to the w3school online editor. Seemed to work.